unbxdAutosuggest.css

```diff
@@ -303,3 +303,44 @@
 .view-more-link a, #sli_autocomplete .sli_ac_section, #sli_autocomplete .view-more-link a:hover {
     background-color: #ec6f34 !important;
 }
+
+/** Mobile css **/
+.unbxd-as-popular-product-info.mobile {
+    display: flex;
```

Comment:
have we validated flex CSS on all browsers.

Reply:
If the intended purpose of `display:flex` was to make the children inline, you can just do a `display:inline-block` on them instead to give more browser support

Reply:
Added support for other browsers for flex